src/Socket.cpp

```diff
@@ -30,7 +30,7 @@
 #ifdef HAVE_LIBSSL
 // Copypasted from
 // https://wiki.mozilla.org/Security/Server_Side_TLS#Intermediate_compatibility_.28default.29
```

Comment:
i think this change is for "server" only. so no znc<->ircd connections, only client<->znc connections.

but good question. modern compatibilty will break:
* Java J2ME clients (really? :)
* webadmin in "old" browsers